packages/loadable-adapters/src/federation-stats-plugin/federation-stats-plugin.js

```diff
@@ -91,6 +95,45 @@ export class FederationStatsPlugin {
           }
         }
       );
+
+      const hasSriPlugin = Boolean(
+        compiler.options.plugins &&
+          compiler.options.plugins.find((plugin) => plugin.constructor.name === 'SubresourceIntegrityPlugin')
+      );
+
+      if (hasSriPlugin) {
+        // needs to be optimized
+        const assetIntegrityMap = new Map();
+
+        compiler.hooks.done.tap(PLUGIN_NAME, (stats) => {
+          assetIntegrityMap.clear();
+
+          stats
+            .toJson()
+            .assets.filter((asset) => Boolean(asset.integrity))
+            .forEach((asset) => {
+              const integrity = Array.isArray(asset.integrity) ? asset.integrity[0] : asset.integrity;
+              console.log(asset.name, integrity);
```

Comment:
less console in lib

Reply:
In [this PR](https://github.com/valorkin/mf-loadable-adapter/pull/3/files) I fixed it, to avoid merge conflicts